_component_design/cards.md

```diff
@@ -149,15 +116,19 @@ Cards are applied to the [Helix grid](layouts/grid) and have a variable width an
 <div class="hxRow"  markdown="1">
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Offset
+### Spacing
 
-- In instances where only few cards are present, a designer may choose to offset the location of the first card in a row for aesthetic purposes. For more technical information, see [the HelixUI grid](https://rackerlabs.github.io/helix-ui/components/grid/).
+Each section of a card uses medium container spacing, 20px, by default.
+
+Small and large spacing values can be used for card variations, but should not
```

Comment:
You can use small and large spacing values for card variations, but do not mix and match spacing values within a single card grid layout or individual card.